plugin-energy-estimation-tdp/README.md

```diff
@@ -0,0 +1,75 @@
+# energy estimation tdp plugin
```

Comment:
```suggestion
# Energy estimation TDP plugin <!-- omit in toc -->
```

Reply:
Toc is for table of content

Reply:
This flag prevent the vscode extension we are using to add these titles in the table of content